### PR TITLE
Us1.2 daily facts component #6

### PR DESCRIPTION
**Pull request by PaulKip44**
Add daily fact list component

### CODE REVIEW

**Review by ClemDSC — APPROVED** [review]
update into different json files and add US files
review ok

File: package.json [hunk 1]
```diff
@@ -17,5 +17,8 @@
   "devDependencies": {
     "concurrently": "^7.0.0",
     "husky": "^7.0.0"
+  },
+  "dependencies": {
+    "prop-types": "^15.8.1"
```
Comment (lchastanet) [inline package.json:22]: This is useless here

**Review by lchastanet — CHANGES_REQUESTED** [review]
Fix the little change please.

**Review by lchastanet — APPROVED** [review]
Ok 👌🏻

**Pull request merged**